content/types/_push_channel.textile

```diff
@@ -13,10 +13,10 @@ bq(definition).
 Subscribe your device to the channel's push notifications.
 
 h6(#subscribe-client-id).
-  default: subscribeClientId
+  default: subscribeClient
```

Comment:
should we update the anchors as well? ie `#subscribe-client-id` -> `#subscribe-client`